data/map.txt

```diff
@@ -27280,7 +27280,7 @@ planet "Wyvern Station"
 
 planet Zenith
 	attributes pirate "north pirate"
-	landscape land/water4
+	landscape land/sea14-sfiera
```

Comment:
This image doesn't really match the description of a world with almost no sunlight and small hidden settlements.